lib/widget/cards/ppe_step_card.dart

```diff
@@ -7,9 +7,10 @@ import '../../widget/cards/reusable_card_base.dart';
 
 class PPECard extends StatelessWidget {
   final PPEStepInfo step;
+  final int index;
   final Color backgroundColor;
 
-  const PPECard({this.step, this.backgroundColor});
+  const PPECard({this.index, this.step, this.backgroundColor});
```

Comment:
why can't the index be a property on the step object?

Reply:
I wanted to make it dynamic, so that index is calculated based on the objects position inside the list